aio/content/examples/dynamic-component-loader/src/app/hero-job-ad.component.ts

```diff
@@ -1,17 +1,17 @@
+/* eslint-disable @angular-eslint/no-input-rename */
 // #docregion
 import { Component, Input } from '@angular/core';
 
-import { AdComponent } from './ad.component';
-
 @Component({
+  standalone: true,
   template: `
     <div class="job-ad">
-      <h4>{{data.headline}}</h4>
-      {{data.body}}
+      <h4>{{ headline }}</h4>
+      {{ body }}
     </div>
-  `
+  `,
 })
-export class HeroJobAdComponent implements AdComponent {
-  @Input() data: any;
+export class HeroJobAdComponent {
+  @Input({ required: true }) headline!: string;
+  @Input({ required: true }) body!: string;
```

Comment:
I think we can drop required flag to keep things simple (sorry, I remember that I've proposed adding them earlier...).
```suggestion
  @Input() headline!: string;
  @Input() body!: string;
```